src/content/roadmap/account-abstraction/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Account abstraction
+description: An overview of Ethereum's plans to make user accounts simpler and safer
+lang: en
+template: staking
+emoji: ":money_with_wings:"
+image: ../../../assets/staking/leslie-pool.png
+alt: Leslie the rhino swimming in the pool.
+sidebarDepth: 2
+summaryPoints:
+  - Understand why account abstraction will make Ethereum easier to use
+  - Learn about upcoming changes to how users will interact with Ethereum
+  - Read why account abstraction will help users to secure their assets
```

Comment:
Summary points, when used elsewhere on the site, give the high-level summary of the  contents of the page. This is being used to tell users what they will learn (if they read the entire page), without actually giving them any of the information. 

Was this a conscious choice? If so, would be curious to hear the rational here.

cc: @konopkja

Reply:
To be more specific, I'd expect this to say something like:

  - Account abstraction will make Ethereum easier to use
  - etc
  - etc